lib/nimbase.h

```diff
@@ -439,15 +440,19 @@ struct TFrame_ {
 #ifdef NIM_NEW_MANGLING_RULES
   #define nimfr_(proc, file) \
     TFrame FR_; \
-    FR_.procname = proc; FR_.filename = file; FR_.line = 0; FR_.len = 0; nimFrame(&FR_);
+    FR_.procname = proc; FR_.filename = file; FR_.line = 0; FR_.col = 0; FR_.len = 0; nimFrame(&FR_);
 
   #define nimfrs_(proc, file, slots, length) \
-    struct {TFrame* prev;NCSTRING procname;NI line;NCSTRING filename; NI len; VarSlot s[slots];} FR_; \
-    FR_.procname = proc; FR_.filename = file; FR_.line = 0; FR_.len = length; nimFrame((TFrame*)&FR_);
+    struct {TFrame* prev;NCSTRING procname;NI line;NI col;NCSTRING filename; NI len; VarSlot s[slots];} FR_; \
+    FR_.procname = proc; FR_.filename = file; FR_.line = 0; FR_.col = 0; FR_.len = length; nimFrame((TFrame*)&FR_);
 
-  #define nimln_(n, file) \
-    FR_.line = n; FR_.filename = file;
-#else
+  // keeping this for bootstrapping
+  #define nimln_(line2, file) \
+    FR_.line = line2; FR_.filename = file;
+
+  #define nimln2_(line2, col2, file) \
+    FR_.line = line2; FR_.col = col2; FR_.filename = file;
+#else // todo: time to cleanup this branch?
```

Comment:
note to reviewer: can this else branch be removed? (ie just keep NIM_NEW_MANGLING_RULES)

Reply:
Yes but not in this PR.